docs/tutorial/tutorial.md

```diff
@@ -98,6 +100,8 @@ return React.createElement('div', {className: 'shopping-list'},
 );
 ```
 
+https://facebook.github.io/react/docs/react-api.html#createelement
```

Comment:
Just placing a link like this is not very helpful. Is there a better way we could do it? I would also prefer that we do this consistently for all API rather than in one place.